### ISSUE TITLE
deploy_hed_portal_org_settings and deploy_hed_portal tasks undefined ?

### ISSUE THREAD

**Issue by RupertBarrow** [SalesforceFoundation/SAL-CumulusCI#2]
Hi, I'm trying to understand the process of deploying an ExperienceBundle with CCI.

Have you forgotten to define the deploy_hed_portal_org_settings  and deploy_hed_portal tasks mentioned in this repo's README ?

**Comment by davidjray** [comment]
Hi @RupertBarrow thanks for bringing this to our attention. The product team will be addressing the issues you asked about here in an upcoming fix.

- The reference to `deploy_hed_portal` looks like it has been removed already
- The reference to `deploy_hed_portal_org_settings` is getting removed soon.

These references are not needed do deploy the metadata included here. Internally we do development with Experience Bundle Metadata, but for compatibility reasons we must use the older `sites` metadata type. I certainly recommend using Experience bundles otherwise.

If you would like to use experience bundles with CumulusCI the orgs you are working with require the ExperienceBundle metadata type to be enabled. Then you can retrieve and deploy the metadata like usual.

To enable ExperienceBundle metadata types in scratch orgs the org definition file would look similar to this:
```
{
  "orgName": "Sample Org",
  "edition": "developer",
  "features": [
    "COMMUNITIES"
  ],
  "settings": {
    "experienceBundleSettings": {
      "enableExperienceBundleMetadata": true
    },
    "communitiesSettings": {
      "enableNetworksEnabled": true
    }
  }
}
```

**Comment by davidjray** [comment]
Fixed by 5214f852
